game/selenium_tests/base.py

```
@@ -0,0 +1,93 @@
+
+        chrome_options = Options()
+        chrome_options.add_argument("--headless")
+        chrome_options.add_argument("--no-sandbox")
+        chrome_options.add_argument("--disable-dev-shm-usage")
+        chrome_options.add_argument("--disable-gpu")
+        chrome_options.add_argument("--window-size=1920,1080")
+
+        try:
+            cls.driver = webdriver.Chrome(options=chrome_options)
+            log_ok("Chrome WebDriver initialized")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)